web/business-registry-dashboard/app/components/Modal/ManageBusiness.vue

```diff
@@ -174,17 +178,27 @@ onMounted(async () => {
           {{ $t('form.manageBusiness.missingInfo.fragmentPrt3') }}
         </p> -->
 
-        <!-- temporarily disabled per #29292
-        <p>The business doesn't have an email on file. Please contact B.C. Registries by choosing one of the options in the help section below</p> -->
-
-        <!-- temporary text per #29292: -->
-        <p>
-          Your company is still managed through <a href="https://www.corporateonline.gov.bc.ca" target="_blank" class="text-blue-500 underline">Corporate Online</a>, and will be moved to the BC Business Registry soon.
-        </p>
-
-        <p>
-          To keep up to date on when businesses can be self-managed through the BC Business Registry application, visit and subscribe to <a href="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization" target="_blank" class="text-blue-500 underline">http://bcreg.ca/updates</a>.
-        </p>
+        <!-- temporarily disabled per #29789 -->
+        <!-- company is in LEAR but doesn't have an email on file -->
+        <div v-if="isLearBusiness">
```

Comment:
If this doesn't need to be a div for any reason (eg, styling) then please change it to a `template`.

Ditto for the else block.